packages/backend/src/telemetry/service.ts

```diff
@@ -85,6 +87,24 @@ class TelemetryServiceImpl implements TelemetryService {
     await this.meterProvider?.shutdown()
   }
 
+  private createHistogram(name: string, options: MetricOptions | undefined) {
+    const histogram = metrics
+      .getMeter(METER_NAME)
+      .createHistogram(name, options)
+    this.histograms.set(name, histogram)
+    return histogram
+  }
+  public getOrCreateHistogramMetric(
```

Comment:
the analogous one for our counters is named `getOrCreateMetric`. So it implicitly assumed metrics were counters (which was true up until adding this). I didn't touch that one but considered renaming it to `getOrCreateCounterMetric`.

Reply:
I agree with renaming for clarity.

Reply:
going to hold off on this renaming because this will change with https://github.com/interledger/rafiki/issues/2799